extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -345,6 +347,7 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
         this.responsiveState_ != null
           ? this.responsiveState_.getRafmtParam()
           : null,
+      'gdpr': gdprApplies === true ? '1' : gdprApplies === false ? '0' : null,
```

Comment:
To clarify, a corner case is 
`gdpr_consent` contains the U.S. privacy string, and the endpoint doesn't have `gdprApplies`, and the fallback `consentRequired` value is used. 
So you would have both `gdpr`=1 and a `gdpr_consent` U.S. privacy string value. 

I want to double check that you're aware of this and it will be handled correctly.

Reply:
Discussed offline.

Reply:
It looks this is resolved. If not, feel free to reopen!